pkg/moq/testpackages/typealiaspkgimport/message/message.go

```diff
@@ -0,0 +1,5 @@
+package message
+
+import "github.com/matryer/moq/pkg/moq/testpackages/typealiaspkgimport/internal/message"
+
+type Message = message.Message
```

Comment:
You're importing a package with the same name as the current package. I wonder if that matters. Does the issue still manifest if you do e.g.

```suggestion
package message

import internalmessage "github.com/matryer/moq/pkg/moq/testpackages/typealiaspkgimport/internal/message"

type Message = internalmessage.Message
```

Reply:
I have changed the name where the type alias was present to `alias`. The same issue is [still present](https://github.com/matryer/moq/pull/155/checks?check_run_id=3411327955#step:8:19).

Reply:
Aw beans.